src/shared/options.ts

```diff
@@ -430,6 +431,9 @@ class RazorOptionsImpl implements RazorOptions {
     public get razorServerPath() {
         return readOption<string>('razor.languageServer.directory', '');
     }
+    public get cohostingEnabled() {
```

Comment:
I assume this option requires a reload?  If so we should add it to the needs reload options

Reply:
I think all Razor options currently need a reload :)